### ISSUE TITLE
Crash: inquirer.prompt is not a function

### ISSUE THREAD

**Issue by michael-benrachel** [garden-io/garden#6640]
### Error message

```
TypeError: inquirer.prompt is not a function
    at userPrompt (file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:193198:21)
    at async CreateProjectCommand.action (file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:439051:28)
    at async file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:433523:30
    at async file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:433311:20
    at async file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:915518:26
    at async file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:915686:33
    at async file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:433311:20
    at async GardenCli.run (file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:915684:32)
    at async file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:915843:29
    at async file:///Users/michael.benrachel/Library/Application%20Support/io.garden.garden/1731481495-ldR9p8w.r/rollup/garden.mjs:433311:20
```

### What did you do?

I ran garden create project as part of the tutorial at https://docs.garden.io/tutorials/your-first-project/1-initialize-a-project
### Your environment

MacOS Sonoma 14.6.1 on M1 Pro MacbookPro, 32GB
minikube
garden version: 0.13.43

* **OS:** 
* **How I'm running Kubernetes:** 
* **Garden version:** 

### Frequency

tried 3 times, same error each time

### Workaround

No idea, I am but a tiny garden egg

**Comment by vvagaytsev** [comment]
Hi @michael-benrachel, thanks for reporting this!

We have already fixed that in #6613. We plan to release Garden 0.13.44 with the fix this week.

**Comment by vvagaytsev** [comment]
Closing as a duplicate of #6612.
